etc/org.opencastproject.adminui.cfg

```diff
@@ -231,3 +231,8 @@ sourcetrack.right.flavor=presentation/prepared
 #
 # Default: video+preview
 #preview.video.subtype=video+preview
+
+# ID of the workflow used to unpublish events when deleting.
+#
+# Default: delete
+# retract.workflow.id=delete
```

Comment:
@krinnewitz what is the Use Case for changing this default? Is the alternative to delete from Opencast asset manager/adminUI/adminSearch/adminIndex but not un-publish? 
A possibly unintended benefit of this config item is that it clearly identifies which workflow can be adjusted to change the default retractions.

Reply:
> @krinnewitz what is the Use Case for changing this default? Is the alternative to delete from Opencast asset manager/adminUI/adminSearch/adminIndex but not un-publish?
> A possibly unintended benefit of this config item is that it clearly identifies which workflow can be adjusted to change the default retractions.

@karendolan I don't have a specific use case in mind, but the alternative to making it configurable would be to hard-code the workflow id. I think by making it configurable, we have maximum flexibility as well as the documental benefit you mentioned. However, I agree that changing the workflow ID is a rare use case (if any at all) and people would rather edit the workflow itself.

Reply:
@karendolan & @krinnewitz Our use case is that we use a naming scheme where the workflow definition identifier includes both a prefix to clearly identify SWITCH custom workflow definitions and a workflow definition version number (e.g. switch-delete-1.0).
Besides that, I strongly dislike the idea of hard-coded identifiers ;-)